src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -351,7 +350,7 @@ protected override async Task OnDeleteAsync(Uri resourceUri, DeleteRequestOption
         {
             string name = GetName(resourceUri);
 
-            var accessCondition = (deleteRequestOptions as DeleteRequestOptionsWithAccessCondition)?.AccessCondition;
+            var accessCondition = (deleteRequestOptions as DeleteRequestOptionsWithAccessCondition)?.AccessCondition ?? null;
```

Comment:
wait isn't `?? null` kind of a no-op?

Reply:
Same comment for this line: https://github.com/NuGet/NuGet.Services.Metadata/pull/549/files#diff-7d744c5a2a0334b0e4d8c4c000aa6469R229

Reply:
lmao good catch